pkg/http/api.go

```diff
@@ -1134,7 +1134,7 @@ func (a *api) onGetMetadata(reqCtx *fasthttp.RequestCtx) {
 		activeActorsCount = a.actor.GetActiveActorsCount(reqCtx)
 	}
 
-	registeredComponents := []registeredComponent{}
+	registeredComponents := make([]registeredComponent, 0, len(a.getComponentsFn()))
```

Comment:
We can call `a.getComponentsFn()` only once, because there are slice allocation and copy every time it is called.

Reply:
Sorry, my mistake. I changed that line. Calling len using components variable.